expression/util.go

```diff
@@ -273,3 +274,84 @@ func Contains(exprs []Expression, e Expression) bool {
 	}
 	return false
 }
+
+// ExtractFiltersFromDNFs checks whether the cond is DNF and extract the same expression in its leaf items and return them as a slice.
```

Comment:
The comment is not correct. This function not only extract&return common conditions.